CONTRIBUTING.md

```diff
@@ -524,3 +552,6 @@ Emeritus:
 
 See the [community membership document in OpenTelemetry community
 repo](https://github.com/open-telemetry/community/blob/main/community-membership.md).
+
+[Approver]: https://github.com/orgs/open-telemetry/teams/go-approvers
+[Maintainer]: https://github.com/orgs/open-telemetry/teams/go-maintainers
```

Comment:
I think these pages are only accessible to members of the organization.

Reply:
Sure. Is there a better link to use?

Reply:
`CODEOWNERS`  file.

Approvers: https://github.com/open-telemetry/opentelemetry-go/blob/b62eb2ca88dfccbffb729f2213a2146ee76dc34e/CODEOWNERS#L15

Maintainers: https://github.com/open-telemetry/opentelemetry-go/blob/b62eb2ca88dfccbffb729f2213a2146ee76dc34e/CODEOWNERS#L17

Reply:
I updated to link to the listing of approves and maintainers in the CONTRIBUTING.md doc itself. PTAL